src/mmw/js/src/core/views.js

```diff
@@ -2,7 +2,7 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    _ = require('lodash'),
+    _ = require('underscore'),
```

Comment:
Curious why this was changed. Every function that's in underscore should have a corollary in lodash.